### PR TITLE
Policies docs

### PR DESCRIPTION
**Pull request by nl0**

### CODE REVIEW

**Comment by nl0** [comment]
@akarve please edit

File: docs/Catalog/Admin.md [hunk 1]
```diff
@@ -19,23 +19,51 @@ functions:
...
 
-You may create custom roles for different groups of users. With the exception of
-administrators, users of managed roles are only aware of the buckets that they
-are permitted to read. All other stack buckets are invisible to users of managed
-roles.
+You must select the default role for all new users, else they will not be able
+to sign in to the Quilt catalog. The default role is shown in bold.
 
-![](../imgs/admin-users-roles-create-managed.png)
+![](../imgs/admin-set-default-role.png)
```
Comment (akarve) [inline docs/Catalog/Admin.md:31]: I believe this URI has changed per my last docs update (just merged)

File: docs/Catalog/Admin.md [hunk 2]
```diff
@@ -19,23 +19,51 @@ functions:
...
 
-You must select the default role for all new users, else they will
-not be able to sign in to the Quilt catalog.
-The default role is shown in bold.
+You may create roles for different groups of users by combining up to 5 policies.
+With the exception of administrators, users of managed roles are only aware
+of the buckets that they are permitted to read.
```
Comment (akarve) [inline docs/Catalog/Admin.md:35]:
```suggestion
With the exception of administrators, users of managed roles can only see, list, and
search buckets for which they are explicitly granted read access.
```

File: docs/Catalog/Admin.md [hunk 3]
```diff
@@ -19,23 +19,51 @@ functions:
...
+All other stack buckets are invisible to users of managed roles.
```
Comment (akarve) [inline docs/Catalog/Admin.md:36]: Can eliminate this line, it's now redundant.

**Review by akarve — APPROVED** [review]

**Pull request merged**